mindsdb/integrations/handlers/openai_handler/helpers.py

```diff
@@ -13,12 +13,19 @@
 from mindsdb.integrations.handlers.openai_handler.constants import OPENAI_API_BASE
 
 
+class PendingFT(openai.OpenAIError):
```

Comment:
Introduced new class in `wait_errors` as otherwise the base `OpenAIError` would trigger an exception instead of running a new iteration of the exponential backoff when finetuning Anyscale models.